resources/examples/simple_cluster/README

```diff
@@ -1,7 +1,9 @@
 This examples is composed of:
 
-1 management node
-1 login node
-4 computes nodes
+* 1 management node
+* 1 login node
+* 4 computes nodes
 
 All of them are link with a single ethernet management network, and with an interconnect.
```

Comment:
Can you add here the IP address plan of the networks ?

Reply:
I added a simple plan 😊